app/src/main/java/com/kickstarter/libs/AttributionEvents.kt

```diff
@@ -0,0 +1,52 @@
+package com.kickstarter.libs
+
+import android.annotation.SuppressLint
+import com.kickstarter.libs.utils.AnalyticEventsUtils
+import com.kickstarter.libs.utils.ContextPropertyKeyName
+import com.kickstarter.libs.utils.EventContextValues
+import com.kickstarter.libs.utils.EventName
+import com.kickstarter.models.Project
+import com.kickstarter.services.ApolloClientTypeV2
+import com.kickstarter.services.mutations.CreateAttributionEventData
+import com.kickstarter.services.transformers.encodeRelayId
+import com.kickstarter.ui.data.ProjectData
+import timber.log.Timber
+
+/**
+ * Similar to the [AnalyticEvents] class but specifically for sending attribution events to KSR
+ * event attribution backend.
+ *
+ */
+class AttributionEvents(
+    val apolloClient: ApolloClientTypeV2
+) {
+    /**
+     * Sends data to the backend for event attribution when the projects screen is loaded.
+     *
+     * @param projectData: The selected project data.
+     */
+    fun trackProjectPageViewed(projectData: ProjectData) {
+        val props: HashMap<String, Any> = hashMapOf(ContextPropertyKeyName.CONTEXT_PAGE.contextName to EventContextValues.ContextPageName.PROJECT.contextName)
+        props["context_page_url"] = projectData.fullDeeplink().toString()
```

Comment:
Per product requirements, we send the full deeplink, not just the ref tag https://kickstarter.atlassian.net/wiki/spaces/NT/pages/2605875227/Event+Attribution+Spike#Adding-support-for-tracking-the-entire-deep-link-url